clusters/build-clusters/01_cluster/ci/_origin-release-build/nodejs-16/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.ci.openshift.org/openshift/nodejs-16:latest
```

Comment:
Add @jhadvig Not sure if the `ngnix` should be added here too?

Reply:
@cyril-ui-developer you mean do have a separate Dockerfile for it ? and in it create the missing directories ?

I dont actually think that it will work. We should add some debugging steps into the Dockerfile to help us figure out whats happening